hashtheplanet/resources/resource.py

```diff
@@ -12,8 +15,19 @@ class Resource(): # pylint: disable=too-few-public-methods
     def __init__(self, database: DbConnector):
         self._database = database
 
-    def compute_hashes(self, session_scope, target: str):
+    def compute_hashes(self, session_scope, target: str, exclude_regex: Optional[str]):
         """
         This method computes all the versions and their associated files & hashes and stores them in the database.
         """
         raise NotImplementedError()
+
+    @staticmethod
+    def can_save(exclude_regex: str, file_path: str):
```

Comment:
"Can be saved to the database" is inaccurate : all the paths can, we just don't want them! "Should be saved" could be a  better description